helm/charts/infra/templates/NOTES.txt

```diff
@@ -37,6 +37,14 @@
 
 {{- end }}
 
+{{- if and .Values.server.persistence.enabled (not .Values.server.config.dbEncryptionKey) .Release.IsUpgrade }}
+
+  When upgrading from chart version <= 0.14.4, ensure the database encryption
```

Comment:
If the key is not migrated what ends up happening until they migrate it?

Reply:
The new deployment will be unhealthy and the new server will not be usable

Reply:
Maybe we can call out what to do if they miss performing the manual upgrade steps?

Reply:
That's the intention with the link. The upgrade steps are too involved to include in the NOTES so a link to the README is the next best thing